internal/auth/repository/eventsourcing/eventstore/auth_request.go

```diff
@@ -117,18 +118,23 @@ func (repo *AuthRequestRepo) CreateAuthRequest(ctx context.Context, request *dom
 		return nil, err
 	}
 	request.ID = reqID
-	app, err := repo.View.ApplicationByClientID(ctx, request.ApplicationID)
+	//DISCUSS: the 3 following calls refer to the query side but we are on the command side
```

Comment:
👍  we should add a todo for auth request refactoring / login improvements
Spontaneously I'd suggest moving it to the oidc pkg in the api layer

Reply:
added a TODO in `internal/api/oidc/auth_request.go`

Reply:
do we have a tracking issue for that refactoring? I received a 418 from https://livios.brain/caos/zitadel/issues?name=auth%20request%20refactoring  🥴